### ISSUE TITLE
Autoprefixer duplicates `-ms-interpolation-mode` declaration

### ISSUE THREAD

**Issue by mathiasbynens** [postcss/autoprefixer#632]
Given the following input:

```css
canvas {
	-ms-interpolation-mode: nearest-neighbor; /* IE */
	image-rendering: -webkit-optimize-contrast; /* Old Android, Safari, UC Browser */
	image-rendering: -moz-crisp-edges; /* Firefox */
	image-rendering: crisp-edges; /* Safari */
	image-rendering: pixelated; /* Chrome & Opera */
}
```

Actual Autoprefixer output (with default settings):

```css
canvas {
	-ms-interpolation-mode: nearest-neighbor; /* IE */
	image-rendering: -webkit-optimize-contrast; /* Old Android, Safari, UC Browser */
	image-rendering: -moz-crisp-edges; /* Firefox */
	image-rendering: crisp-edges; /* Safari */
	-ms-interpolation-mode: nearest-neighbor;
	    image-rendering: pixelated; /* Chrome & Opera */
}
```

Note that `-ms-interpolation-mode: nearest-neighbor;` is now duplicated.

Simple test case to reproduce the issue:

```js
// `npm i autoprefixer postcss`
'use strict';
const autoprefixer = require('autoprefixer');
const postcss = require('postcss');

const plugin = autoprefixer();

const css = `
canvas {
	-ms-interpolation-mode: nearest-neighbor; /* IE */
	image-rendering: -webkit-optimize-contrast; /* Old Android, Safari, UC Browser */
	image-rendering: -moz-crisp-edges; /* Firefox */
	image-rendering: crisp-edges; /* Safari */
	image-rendering: pixelated; /* Chrome & Opera */
}
`;

postcss([ autoprefixer ]).process(css).then(function(result) {
	result.warnings().forEach(function(warn) {
		console.warn(warn.toString());
	});
	console.log(result.css);
});
```

**Comment by ai** [comment]
Thanks. I will try to fix it on this weekend.

**Comment by ai** [comment]
Done 01476c9

**Comment by ai** [comment]
Released in 6.3.5
